src/core/server/http/router/socket.ts

```diff
@@ -52,15 +67,14 @@ export interface IKibanaSocket {
 }
 
 export class KibanaSocket implements IKibanaSocket {
-  readonly authorized?: boolean;
-  readonly authorizationError?: Error;
-
-  constructor(private readonly socket: Socket) {
-    if (this.socket instanceof TLSSocket) {
-      this.authorized = this.socket.authorized;
-      this.authorizationError = this.socket.authorizationError;
-    }
+  public get authorized() {
+    return this.socket instanceof TLSSocket ? this.socket.authorized : undefined;
   }
+  public get authorizationError() {
+    return this.socket instanceof TLSSocket ? this.socket.authorizationError : undefined;
+  }
+
+  constructor(public readonly socket: Socket) {}
```

Comment:
Do we intentionally make the socket public (it was private)?

Reply:
😬  Nope, that was intended to be a temporary change while I was testing how renegotiate worked. Thanks for catching, I'll change that back.